pkg/bootstrap/config.go

```diff
@@ -59,26 +59,21 @@ const (
 // Config for creating a bootstrap file.
 type Config struct {
 	*model.Node
-	PilotSubjectAltName []string
-	ProxyViaAgent       bool
-	OutlierLogPath      string
-	PilotCertProvider   string
-	ProvCert            string
-	DiscoveryHost       string
 }
 
 // newTemplateParams creates a new template configuration for the given configuration.
 func (cfg Config) toTemplateParams() (map[string]interface{}, error) {
 	opts := make([]option.Instance, 0)
 
+	discHost := strings.Split(cfg.Metadata.ProxyConfig.DiscoveryAddress, ":")[0]
```

Comment:
nit: should we use net.SplitHostPort? I don't think it matters here since we don't allow `https://foo` anyways though so maybe not important

Reply:
I see this is copied from somewhere else, so consider it a super nitpick...